pkg/registries/docker/metadata_v2.go

```diff
@@ -13,12 +13,13 @@ func HandleV2ManifestList(r *Registry, remote, ref string) (*storage.ImageMetada
 		return nil, err
 	}
 	if len(manifestList.Manifests) == 1 {
-		return HandleV2Manifest(r, remote, manifestList.Manifests[0].Digest)
+		return HandleV2Manifest(r, remote, manifestList.Manifests[0].Digest.String())
```

Comment:
If its possible for `.Digest` to be nil now as a result should probably guard that statement with a nil check to avoid a panic, if not possible than /LGTM!

Reply:
`Digest` is just an alias for `string` https://github.com/opencontainers/go-digest/blob/master/digest.go#L37